ksqldb-engine/src/main/java/io/confluent/ksql/internal/PullQueryExecutorMetrics.java

```diff
@@ -132,15 +215,15 @@ public Metrics getMetrics() {
   private Sensor configureLocalRequestsSensor() {
     final Sensor sensor = metrics.sensor(
         PULL_QUERY_METRIC_GROUP + "-" + PULL_REQUESTS + "-local");
-    
+
     // legacy
     addSensor(
         sensor,
         PULL_REQUESTS + "-local-count",
         ksqlServiceIdLegacyPrefix + PULL_QUERY_METRIC_GROUP,
         "Count of local pull query requests",
         legacyCustomMetricsTags,
-        new WindowedCount()
+        new CumulativeCount()
```

Comment:
Yeah, they are a bit piggybacked on this since I'm assuming that we should fix existing metrics.  Anything in this class that used `WindowedCount` is essentially wrong as far as I can tell.

All of the windowed functions are a SampledStat, and used directly, they use the configs set up in `MetricCollectors` which have this snippet:

```
final MetricConfig metricConfig = new MetricConfig()
        .samples(100)
        .timeWindow(
            1000,
            TimeUnit.MILLISECONDS
        );
```

What this means is that it keeps 100 seconds of samples around.  The value reported for this metric is then just effectively the count for the last 100 seconds, an entirely arbitrary and not useful value.  This type of sampled stat is much more meaningful if you're reporting an average, not so much a count, since it's not even monotonically increasing.

We still actually have a rate metric defined right below this one.  These are much more useful for reporting to a monitor.

When it comes to other metrics pipelines, if you want to accumulate a count over an arbitrary time, it's easier to use a monotonically increasing counter (and just subtract two data points) rather than integrating over a rate metric (though that's also possible to do).  For this reason, we report both.